auth/contract/contract_test.go

```diff
@@ -222,4 +223,12 @@ func TestParseContractString(t *testing.T) {
 		assert.NotNil(t, err)
 		assert.True(t, errors.Is(err, ErrInvalidContractText))
 	})
+	t.Run("ok", func(t *testing.T) {
```

Comment:
I would make this explicitly an ampersand test and rename it to something like: "ok - it parses a contract text containing ampersand characters"